ch_backup/logic/lock_manager.py

```diff
@@ -106,7 +108,17 @@ def _zk_flock(self) -> None:
             if not client.exists(self._process_zk_lockfile_path):
                 client.create(self._process_zk_lockfile_path, makepath=True)
             self._zk_lock = client.Lock(self._process_zk_lockfile_path)
-            if not self._zk_lock.acquire(blocking=False):
+            try:
+                lock_acquired = self._zk_lock.acquire(
```

Comment:
Do we need `lock_acquired` variable ? Why not just call exit in an `except` block ?